tests/gold_tests/pluginTest/xdebug/x_remap/out.gold

```diff
@@ -37,9 +37,11 @@ X-Remap: from=http://one/, to=http://127.0.0.1:SERVER_PORT/
 ``
 {'xDebugProbeAt' : '``
    'captured':[{'type':'request', 'side':'client', 'headers': {
+	'Start-Line' : 'GET http://127.0.0.1:SERVER_PORT/argh HTTP/1.1',
```

Comment:
This is really called a Request-Line, but I don't think it really matters that much.